domain/src/main/java/jinsu/antilog/domain/Operator.kt

```diff
@@ -0,0 +1,34 @@
+package jinsu.antilog.domain
+
+sealed class Operator {
+    object Plus : Operator() {
+        override fun operate(left: Double, right: Double): Double = left + right
+    }
+
+    object Minus : Operator() {
+        override fun operate(left: Double, right: Double): Double = left - right
+    }
+
+    object Multiply : Operator() {
+        override fun operate(left: Double, right: Double): Double = left * right
+    }
+
+    object Divide : Operator() {
+        override fun operate(left: Double, right: Double): Double {
+            if (right == 0.0) throw IllegalArgumentException("0으로는 나눌 수 없습니다.")
```

Comment:
require() {} 를 활용해보는 것은 어떨까요?
```suggestion
            require(right != 0.0) { "0으로는 나눌 수 없습니다." }
```